TESTING.md

```diff
@@ -531,3 +532,14 @@ Repeat the above starting from different themes.
 #### Expected behaviour
 
 Changing theme should lead to full repaint of the colours according to the current theme.
+
+
+### Claude 3 Enterprise Testing
+#### Cody Gateway Instance
+- [ ] Ask Cody a question in a new chat window. The question should include a request for Cody to generate code with the site-admin configuration is set to **Claude 3 Sonnet** for chatModel and **Claude 3 Haiku** for completionModel
```

Comment:
I do not understand how tester can know/ensure this?

Reply:
Is it better/easier if we just say 'Ask Cody a question in a new chat window with `Instance_name`?

Reply:
@aramaraju Just want to clarify, `Instance_name` is one we'd set up with chat and completions configured with those models? That sounds good to me... and we should put instructions on how to get credentials, the exact endpoint to log in to. This will keep the test turnaround time as fast as possible, increase confidence that we're covering what we want to test, and make the results useful because anyone can repro.

Reply:
@dominiccooney Yes, that is my suggestion, specifically the exact endpoint. We can provide the instance as I'm inferring from Piotr's comment that the QA team doesn't actually configure changes on the instance. We can also find a way to provide the credentials for them in place

Reply:
I'll reiterate, that sounds good to me.

Reply:
I don't think we need to QA this part, our QA testers should assume the environment we give them have the default configurations they need, unless we expand the scope to have them do configuration as well - which I don't recommend at the moment because it means we need to onboard them as a Sourcegraph team member. 

This QA step is actually something a Sourcegraph team should own, as it touches configuration of the instance. As a client team, I think we should focus on the feature test cases instead. I think eventually we will want to expand the QA scope to include Sourcegraph as well, but at the moment we can't.

Reply:
@kalanchan 

> I don't think we need to QA the part around different models, our QA testers should assume the environment we give them have the default configurations they need, unless we expand the scope to have them do configuration as well

Everyone is in agreement that testers should not be having to edit admin policy.

Question for @kalanchan / @aramaraju ...  the updated diff now mentions different instances. Can we mention where the creds for these instances are in this file? Obviously the details of the creds won't be accessible to open source contributors, etc. But we need to drop a breadcrumb so teammates and contractors know where to get those creds, endpoint URLs, etc.